Source/CarthageKit/Netrc.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+import Result
+
+internal struct NetrcMachine {
```

Comment:
You don't need to mark this as internal. It's internal by default

Reply:
Hey @tmspzz, thank you for your comments. I have implemented the changes, could you please have a look? 👀 🙇

Reply:
I think it should be added to `bootstrap`, `update` and possibly `build`

Reply:
Looks like `bootstrap` command already shares the command options with `update` command: 

https://github.com/Carthage/Carthage/blob/68abf2d73791720457f5be1aab4b57df86483f88/Source/carthage/Bootstrap.swift#L12-L14

Added `--use-netrc` to `build` command 🙇